packages/playwright-test/types/component.d.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+type JsonPrimitive = string | number | boolean | null;
```

Comment:
I'm a little worried as this becomes a subject to the public semver, but then again, entire file does. We should rename it to experimentalComponent.d.ts in a follow-up patch to set expectations.

Reply:
Alright, will do.

Wonder if have you considered a separate folder/package for the shared ct code?